sdk/@launchdarkly/observability-android/lib/src/main/kotlin/com/launchdarkly/observability/client/NoopLogRecordPrcoessor.kt

```diff
@@ -0,0 +1,19 @@
+package com.launchdarkly.observability.client
+
+import io.opentelemetry.context.Context
+import io.opentelemetry.sdk.logs.LogRecordProcessor
+import io.opentelemetry.sdk.logs.ReadWriteLogRecord
+
+/**
+ * A [LogRecordProcessor] that, surprise, does nothing.
+ */
+internal class NoopLogRecordProcessor private constructor() : LogRecordProcessor {
+    override fun onEmit(context: Context, logRecord: ReadWriteLogRecord) {}
+
+    companion object {
+        private val INSTANCE = NoopLogRecordProcessor()
+
+        val instance: LogRecordProcessor
+            get() = INSTANCE
+    }
+}
```

Comment:
You could replace this class by using an object to make it simpler, then you just use it like `NoopLogRecordProcessor` instead of `NoopLogRecordProcessor.instance`
```suggestion
internal object NoopLogRecordProcessor : LogRecordProcessor {
    override fun onEmit(context: Context, logRecord: ReadWriteLogRecord) {}
}
```